### PR TITLE
Don't log price oracle failures

### PR DESCRIPTION
**Pull request by Uxio0**
It's not relevant if a price oracle cannot retrieve the price, as there's already an exception returned for that

- Update price oracle logs from `WARNING` to `DEBUG`

### CODE REVIEW

File: gnosis/eth/oracles/utils.py [hunk 1]
```diff
@@ -32,6 +32,6 @@ def get_decimals(
         BadFunctionCallOutput,
         DecodingError,
     ) as e:
-        error_message = f"Cannot get decimals for token={token_address}"
-        logger.warning(error_message)
-        raise CannotGetPriceFromOracle(error_message) from e
+        message = f"Cannot get decimals for token={token_address}"
```
Comment (moisses89) [inline gnosis/eth/oracles/utils.py:35]: Is this related with the price? I mean, ERC20 should have decimals method.

Comment (Uxio0) [reply gnosis/eth/oracles/utils.py:35]: Yes, but when we are calculating the price we use anything that looks like an ERC20, so I wouldn't log this kind of errors. This method is only used from `oracles`

**Review by moisses89 — APPROVED** [review]
